templates/helpers/format-description.js

```diff
@@ -0,0 +1,28 @@
+const Handlebars = require('handlebars');
+
+module.exports = (description) => {
+  const splitDescription = description.split('\n');
+  let initialDescription;
+  let moreDescription;
+
+  const formattedDescription = splitDescription.map((el) => {
+    return '<p>' + el + '</p>';
+  });
+
+  if (formattedDescription.length > 15) {
+    initialDescription = formattedDescription.slice(0, 14);
+    moreDescription = `
+    <div class="details" aria-expanded="false">
+      <div class="details__summary">More…</div>
+        <div class="details__content">
+          ${formattedDescription.slice(15).join('')}
+        </div>
+      </div>`;
+
+    return new Handlebars.SafeString(
+      `${initialDescription.join('')}${moreDescription}`
+    );
+  } else {
+    return new Handlebars.SafeString(formattedDescription.join(''));
+  }
```

Comment:
This all needs to be done before render time in `json-to-html-data.js`.

The description needs to be split into two fields, the handlebars template needs to then detect the presence of the `moreDescription` field and show the HTML for the "More..." expander if necessary.

It keeps HTML in the templates and all the logic for data transformations in `json-to-html-data.js`.